test/Integration/hoist-vector-transfer-brgemm.mlir

```diff
@@ -0,0 +1,59 @@
+// RUN: tpp-opt %s  | tpp-run -e entry --entry-point-result=void  -print > %t.1
```

Comment:
`tpp-opt` here is unnecessary, as `tpp-run` already runs the default pipeline

Reply:
Correct. Removed `tpp-opt`.